public/app/percona/add-instance/components/AddRemoteInstance/FormParts/AdditionalOptions/AdditionalOptions.tsx

```diff
@@ -74,19 +75,25 @@ const MySQLOptions = ({ form }: { form: FormApi }) => {
 
   return (
     <>
-      <RadioButtonGroupField
-        name="tablestat-options"
-        data-testid="tablestat-options-radio-button-group"
-        defaultValue={selectedValue}
-        options={tablestatOptions}
-        label={Messages.form.labels.additionalOptions.tablestatOptions}
-      />
-      <NumberInputField
-        name="tablestats_group_table_limit"
-        defaultValue={-1}
-        disabled={selectedValue !== TablestatOptionsInterface.custom}
-        validate={platformCoreValidators.containsNumber}
-      />
+      <h4>{Messages.form.labels.additionalOptions.tablestatOptions}</h4>
+      <div className={styles.group}>
+        <RadioButtonGroupField
+          name="tablestat-options"
```

Comment:
Makes sense here, for the other names in snake case, it might be more hassle then worth it.

When I took a look into `AddRemoteInstance.types.ts`, it looks like all of the payloads, and basically the inventory section uses the same names as what returns/requests BE.

Reply:
Ok, good to know. Thanks for looking at it, this was done by someone else a long ago, it's nice to know how it's being managed behind the scenes.